docs/source/explanation/algorithms/segmentation/semantic_segmentation.rst

```diff
@@ -109,7 +160,38 @@ For the `DIS5K <https://xuebinqin.github.io/dis/index.html>`_ we prepared random
 Self-supervised Learning
 ************************
 
-To be added soon
+Self-supervised learning can be one of the solutions if the user has a small data set, but label information is not yet available.
+General self-supervised Learning in academia is commonly used to obtain well-pretrained weights from a source dataset without label information.
+However, in real-world industries, it is difficult to apply because of small datasets, limited resources, or training in minutes.
+
+For these cases, OTX provides improved self-supervised learning recipes that can be applied to the above harsh environments.
+We adapted `DetCon <https://arxiv.org/abs/2103.10957>`_ as our self-supervised method.
+It takes some time to use these self-supervised learning recipes, but you can expect improved performance, especially in small-data regimes.
+
+The below table shows how much performance (mDice) self-supervised methods improved compared with baseline performance on the subsets of Pascal VOC 2012 with three classes (person, car, bicycle).
```

Comment:
If there is no labeled data for Self-SL, so what is the SL baseline? Can you clarify it here?

Reply:
Done